### ISSUE TITLE
Unclarity about default CRS in part 1

### ISSUE THREAD

**Issue by lvdbrink** [opengeospatial/ogcapi-features#700]
In OGC API Features part 1, it is stated several times that the default CRS is CRS84 / CRS84h, but there is also some wording that may be interpreted as a possibility to declare another CRS as the default in another part of the standard. 

Specifically, in [7.1](https://docs.ogc.org/is/17-069r3/17-069r3.html#core-overview):

> A list of coordinate reference systems (CRS) in which geometries may be returned by the server: the first CRS is the default coordinate reference system (in the Core, the default is always WGS 84 with axis order longitude/latitude);

This could be taken to mean that in another part which is not the Core, the default could be something else and could perhaps be declared to be the default by listing it as the first CRS in the list. Nothing in part 2 says this, but there is a possibility of misunderstanding here. 

The text should be clarified as to be unambiguous. Depending on what the correct interpretation is it could be e.g: 

"[...] the first CRS is the default coordinate reference system, that is, WGS 84 with axis order longitude/latitude;"

**Comment by cportele** [comment]
Agreed, the current wording in the informative overview is no longer in sync with the normative text in Part 1 and 2. I don't even think that there is a requirement that the default CRS is the first CRS in the array, just that it is included. And the default is either OGC:CRS84 or OGC:CRS84h, not just in Core. I will check and create a PR.
